docs/source/standard.rst

```diff
@@ -0,0 +1,53 @@
+Card Game Notation Standard
+===========================
+This standard uses TOML_ as notation language.
+
+Header
+-------
+All meta data is defined in the Header.
+This contains information about teams and players.
+Furthermore, the location and datetime is stated.
+
+.. code-block:: toml
+
+    [header]
+    # Timestamp of the starting time in RFC 3339
+    timestamp = 2022-08-06T11:22:00
+    # Location of the game or the server url
+    location = "Hamburg"
+    # Unsigned Integer
+    number_of_teams = 2
+    [[players.player_name_1]]
+    # Unsigned Integer: ID of team starting with 0
+    team = 0
+    [[players.player_name_2]]
+    team = 1
+
+Stack
+-----
+The data contains one mandatory field ``stack`` that is a list of strings representing a card.
+Furthermore it has an optional field ``hands`` that is a table of the players' cards in hand.
+One card can be represented with
+
+.. code-block:: toml
+
+    suit = "[♦♣♥♠]"
+    rank = "[2-10JQKA]"
```

Comment:
what happens when cards do not have suit and rank?
like jokers or special cards?

Reply:
Are they part of a French Set? We can always add other sets as sub class.

Reply:
Jokers are not part of the 52-card deck. They can be black or red or colorless instead of a specific suit.

Reply:
So, I would leave them out for now.